ci.rb

```diff
@@ -183,8 +183,8 @@ def get_limits(t_options, t_client, t_repo)
     test_mode = !(ARGV[0] =~ /false/i)
 
     $logger.info "Querying for updated branches"
-    b.query_branches
     b.query_pull_requests
+    b.query_branches
```

Comment:
Reordered the calls here.  Each of these functions just do two independent things, and the order does not appear to matter at all, as they don't really check anything common between them.  I moved the PRs check first though because now it collects a list of "internal" PRs with their branch and their PR number.  Then when we call for branches next, we have that list ready, and can use that PR number.